src/arch/arm/machine/gic_v3.c

```diff
@@ -12,7 +12,7 @@
  */
 #include <config.h>
 #include <types.h>
-#include <arch/machine/gic_500.h>
```

Comment:
This is a good change, but shouldn't the gic_pl390 driver be changed as well to avoid confusion, since it also targets a spec instead of an implementation?

Reply:
I made the rename